dev/NumberBox/NumberBox.cpp

```diff
@@ -35,11 +36,7 @@ NumberBox::NumberBox()
 {
     __RP_Marker_ClassById(RuntimeProfiler::ProfId_NumberBox);
 
-    // Default values for the number formatter
-    const auto formatter = winrt::DecimalFormatter();
-    formatter.IntegerDigits(1);
-    formatter.FractionDigits(0);
-    NumberFormatter(formatter);
+    NumberFormatter(GetRegionalSettingsAwareDecimalFormatter());
```

Comment:
could this be set as the default value for NumberFormatter dependency property when we register it ? I think we can do that if the formatter object is shareable by multiple NumberBoxes.

Reply:
I actually don't want them to share, because I want devs to be able to change e.g. FractionDigits without impacting all the NumberBoxes in their app.